### ISSUE TITLE
Prevent downloading assets multiple times

### ISSUE THREAD

**Issue by gregbugaj** [gregbugaj/marie-ai#24]
There are assets that are being downloaded multiple times as per this log.  This should be downloaded only once per run.

`ro_sharding='none')
roberta2
2022-04-26 13:18:13 | INFO | models.unilm.trocr.task | Load gpt2 dictionary from https://layoutlm.blob.core.windows.net/trocr/dictionaries/gpt2_with_mask.dict.txt
2022-04-26 13:18:15 | INFO | models.unilm.trocr.task | [label] load dictionary: 50265 types
2022-04-26 13:18:15 | INFO | fairseq.file_utils | https://dl.fbaipublicfiles.com/fairseq/gpt2_bpe/encoder.json not found in cache, downloading to /tmp/tmpgnpzp46x
1042301B [00:00, 2283543.33B/s]
2022-04-26 13:18:16 | INFO | fairseq.file_utils | copying /tmp/tmpgnpzp46x to cache at /home/app-svc/.cache/torch/pytorch_fairseq/e2aab4d600e7568c2d88fc7732130ccc815ea84ec63906cb0913c7a3a4906a2e.0f323dfaed92d080380e63f0291d0f31adfa8c61a62cbcb3cb8114f061be27f7
2022-04-26 13:18:16 | INFO | fairseq.file_utils | creating metadata file for /home/app-svc/.cache/torch/pytorch_fairseq/e2aab4d600e7568c2d88fc7732130ccc815ea84ec63906cb0913c7a3a4906a2e.0f323dfaed92d080380e63f0291d0f31adfa8c61a62cbcb3cb8114f061be27f7
2022-04-26 13:18:16 | INFO | fairseq.file_utils | removing temp file /tmp/tmpgnpzp46x
2022-04-26 13:18:16 | INFO | fairseq.file_utils | https://dl.fbaipublicfiles.com/fairseq/gpt2_bpe/vocab.bpe not found in cache, downloading to /tmp/tmpb9t7i28q
456318B [00:00, 1740809.43B/s]
2022-04-26 13:18:17 | INFO | fairseq.file_utils | copying /tmp/tmpb9t7i28q to cache at /home/app-svc/.cache/torch/pytorch_fairseq/b04a6d337c09f464fe8f0df1d3524db88a597007d63f05d97e437f65840cdba5.939bed25cbdab15712bac084ee713d6c78e221c5156c68cb0076b03f5170600f
2022-04-26 13:18:17 | INFO | fairseq.file_utils | creating metadata file for /home/app-svc/.cache/torch/pytorch_fairseq/b04a6d337c09f464fe8f0df1d3524db88a597007d63f05d97e437f65840cdba5.939bed25cbdab15712bac084ee713d6c78e221c5156c68cb0076b03f5170600f
2022-04-26 13:18:17 | INFO | fairseq.file_utils | removing temp file /tmp/tmpb9t7i28q
2022-04-26 13:18:20 | INFO | models.unilm.trocr.deit_models | Using the learned pos embedding version loading roberta.
2022-04-26 13:18:20 | INFO | models.unilm.trocr.deit_models | Load pre-trained decoder parameters from roberta.large
Downloading: "https://github.com/pytorch/fairseq/archive/main.zip" to /home/app-svc/.cache/torch/hub/main.zip
2022-04-26 13:18:24 | INFO | fairseq.file_utils | http://dl.fbaipublicfiles.com/fairseq/models/roberta.large.tar.gz not found in cache, downloading to /tmp/tmpi8_krgek
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 655283069/655283069 [01:01<00:00, 10669018.59B/s]
2022-04-26 13:19:25 | INFO | fairseq.file_utils | copying /tmp/tmpi8_krgek to cache at /home/app-svc/.cache/torch/pytorch_fairseq/83e3a689e28e5e4696ecb0bbb05a77355444a5c8a3437e0f736d8a564e80035e.c687083d14776c1979f3f71654febb42f2bb3d9a94ff7ebdfe1ac6748dba89d2
2022-04-26 13:19:25 | INFO | fairseq.file_utils | creating metadata file for /home/app-svc/.cache/torch/pytorch_fairseq/83e3a689e28e5e4696ecb0bbb05a77355444a5c8a3437e0f736d8a564e80035e.c687083d14776c1979f3f71654febb42f2bb3d9a94ff7ebdfe1ac6748dba89d2
2022-04-26 13:19:25 | INFO | fairseq.file_utils | removing temp file /tmp/tmpi8_krgek
2022-04-26 13:19:26 | INFO | fairseq.file_utils | loading archive file http://dl.fbaipublicfiles.com/fairseq/models/roberta.large.tar.gz from cache at /home/app-svc/.cache/torch/pytorch_fairseq/83e3a689e28e5e4696ecb0bbb05a77355444a5c8a3437e0f736d8a564e80035e.c687083d14776c1979f3f71654febb42f2bb3d9a94ff7ebdfe1ac6748dba89d2
2022-04-26 13:19:26 | INFO | fairseq.file_utils | extracting archive file /home/app-svc/.cache/torch/pytorch_fairseq/83e3a689e28e5e4696ecb0bbb05a77355444a5c8a3437e0f736d8a564e80035e.c687083d14776c1979f3f71654febb42f2bb3d9a94ff7ebdfe1ac6748dba89d2 to temp dir /tmp/tmpq4hr_lrz
`
